File: 1-easy/problems/1018-binary-prefix-divisible-by-5/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def prefixesDivBy5(self, A):
        """
        :type A: List[int]
        :rtype: List[bool]
        """
        for i in range(1, len(A)):
            A[i] += A[i-1] * 2 % 5
        return [x % 5 == 0 for x in A]

File: 1-easy/problems/1018-binary-prefix-divisible-by-5/solution.ts
function prefixesDivBy5(A: any): boolean | number | string | any {
  """
  :type A: List[int]
  :rtype: List[bool]
  """
  for (i in Array.from({length: 1, (A}, (_, i) => i).length)) {
      A[i] += A[i-1] * 2 % 5
  return [x % 5 == 0 for x in A];
}

export { prefixesDivBy5 };
